asp-deploy/orb.yml

```diff
@@ -0,0 +1,120 @@
+version: 2.1
+description: "Update ASP deployments with gitops."
+
+orbs:
+  aws-ecr: circleci/aws-ecr@7.0.0
+
+commands:
+  update-gitops:
+    parameters:
+      env:
+        type: string
+        description: "[Description] Environment in which the service is being deployed (e.g. prod/nonprod)."
+      service-name:
+        type: string
```

Comment:
Not sure if this will work like this, I don't think circleci will evaluate the env variable you should pass the env name instead